examples/larson_hue.py

```diff
@@ -28,11 +28,11 @@
     # Now we generate a value from 0 to 7
     offset = int(round(offset * 7))
 
-    for x in range(8):
+    for x in range(NUM_PIXELS):
         sat = 1.0
  
-        val = 7 - (abs(offset - x) * FALLOFF)
-        val /= 7.0 # Convert to 0.0 to 1.0
+        val = (NUM_PIXELS-1) - (abs(offset - x) * FALLOFF)
+        val /= ((NUM_PIXELS-1)*1.0) # Convert to 0.0 to 1.0
```

Comment:
While we're tidying things up, this might be made clearer by declaring a `max_val = NUM_PIXELS-1` outside of the loop and then:

```python
val = max_val - (abs(offset - x) * FALLOFF)
val /= float(max_val)
```